<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../OneDrive/Pictures/resource files.qrc">
    <normaloff>:/icons/chevron-left.svg</normaloff>:/icons/chevron-left.svg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget {
    background-color: rgb(255, 255, 200); /* Màu vàng nhạt */
}
QPushButton {
    background-color: rgb(255, 165, 0); /* Màu cam */
    color: white; /* Chữ màu trắng */
    border-radius: 5px;
    padding: 5px;
}

QPushButton:hover {
    background-color: rgb(255, 140, 0); /* Màu cam đậm khi hover */
}</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>Quay lại</string>
         </property>
         <property name="icon">
          <iconset resource="../../OneDrive/Pictures/resource files.qrc">
           <normaloff>:/icons/chevron-left.svg</normaloff>:/icons/chevron-left.svg</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 227, 170)</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:rgb(255, 227, 170)</string>
         </property>
         <property name="text">
          <string>FLASHCARD</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(252, 255, 234);</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Mặt trước</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Mặt sau</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(240, 246, 209);

QWidget {
    background-color: rgb(255, 255, 200); /* Màu vàng nhạt */
}
QPushButton {
    background-color: rgb(255, 165, 0); /* Màu cam */
    color: white; /* Chữ màu trắng */
    border-radius: 5px;
    padding: 5px;
}

QPushButton:hover {
    background-color: rgb(255, 140, 0); /* Màu cam đậm khi hover */
}</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="styleSheet">
          <string notr="true">QWidget {
    background-color: rgb(255, 255, 200); /* Màu vàng nhạt */
}
QPushButton {
    background-color: rgb(255, 165, 0); /* Màu cam */
    color: white; /* Chữ màu trắng */
    border-radius: 5px;
    padding: 5px;
}

QPushButton:hover {
    background-color: rgb(255, 140, 0); /* Màu cam đậm khi hover */
}</string>
         </property>
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="styleSheet">
          <string notr="true">QWidget {
    background-color: rgb(255, 255, 200); /* Màu vàng nhạt */
}
QPushButton {
    background-color: rgb(255, 165, 0); /* Màu cam */
    color: white; /* Chữ màu trắng */
    border-radius: 5px;
    padding: 5px;
}

QPushButton:hover {
    background-color: rgb(255, 140, 0); /* Màu cam đậm khi hover */
}</string>
         </property>
         <property name="text">
          <string>Next</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>775</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../OneDrive/Pictures/resource files.qrc"/>
 </resources>
 <connections/>
</ui>
